frontend/src/components/ViewTrips/trip.js

```diff
@@ -68,6 +74,8 @@ function getTripInfoRow(rowText, icon) {
  * @property {string} props.tripId The document id associated with the trip.
  * @property {Function} props.handleEditTrip Event handler responsible for
  *     displaying the edit trip modal.
+ * @property {TripView} props.tripView The current user's trips page view.
+ * @property {string} props.eventKey ...
```

Comment:
`...`  description.

Reply:
Merged up the PR chain so this is fixed now.